docs/bin/plugin_formatter.py

```diff
@@ -125,12 +125,13 @@ def write_data(text, options, outputname, module):
 #####################################################################################
 
 
-def list_modules(module_dir, depth=0):
+def list_modules(module_dir, depth=0, limit_to_modules="all"):
```

Comment:
I think I would make limit_to_modules arg a list here, and do the comma split on the cli arg before calling list_modules. 

(currently it looks this would do odd things if attempting to build module docs for a module with a name like 'ec2_vps_cwt_jkg_none_facts' or 'firewalld' or 'easy_install'?  Haven't tested though so that may be covered...)